packages/main/src/WheelSlider.js

```diff
@@ -279,28 +270,45 @@ class WheelSlider extends UI5Element {
 	}
 
 	_selectElementByIndex(currentIndex) {
-		const sliderElement = this.shadowRoot.getElementById(`${this._id}--items-list`);
-		const itemsCount = this._itemsToShow.length;
-		const itemCellHeight = this._itemCellHeight ? this._itemCellHeight : 2.875;
-		const offsetStep = isPhone() ? 4 : 2;
 		let index = currentIndex;
+		const itemsCount = this._itemsToShow.length;
+		const scrollBy = 46 * index;
+		const sliderElementsWrapper = this.shadowRoot.querySelector(`#${this._id}--wrapper`);
```

Comment:
Overall this line is repeated often. You should create a getter for it. Moreover, it's simpler to never use IDs, but just classes, as you query inside the shadow root anyway.